Packages/Sites/Neos.NeosIo/Resources/Public/Styles/Main.css

```diff
@@ -531,7 +531,6 @@ input::-moz-focus-inner {
 .u-textBreak {
   word-wrap: break-word;
   -webkit-hyphens: auto;
-     -moz-hyphens: auto;
```

Comment:
I'm unsure about this line and the following (1913). I didn't do anything special with my installed sass packages but maybe something is missing or I have the wrong version?